nullaway/src/main/java/com/uber/nullaway/generics/GenericsChecks.java

```diff
@@ -616,8 +616,48 @@ public static void compareGenericTypeParameterNullabilityForCall(
       // all remaining actual arguments in the next loop.
       n = n - 1;
     }
+
+    List<Symbol.VarSymbol> newFormalParameters = new ArrayList<>();
```

Comment:
I'm pretty sure this logic is not right.  It's looking at upper bounds, which are not relevant for this check.  Just like what we did before for generic methods, we need to look at the _type arguments_ at the call site and apply the substitution to the types of the formal parameters.